spec/facter/resolvers/linux/networking_spec.rb

```diff
@@ -295,6 +297,8 @@
         allow(File).to receive(:exist?).with('/proc/net/if_inet6').and_return(true)
         allow(Facter::Util::FileHelper).to receive(:safe_read)
           .with('/proc/net/if_inet6', nil).and_return(load_fixture('proc_net_if_inet6').read)
+        allow(Facter::Util::FileHelper).to receive(:safe_read).with("/sys/class/net/lo/operstate").and_return('unknown')
+        allow(Facter::Util::FileHelper).to receive(:safe_read).with("/sys/class/net/ens160/operstate").and_return('unknown')
```

Comment:
```suggestion
        allow(Facter::Util::FileHelper).to receive(:safe_read).with("/sys/class/net/ens160/operstate").and_return('up')
```

so that it matches the `result` that we're expecting

Reply:
I thought I'd set `lo` to `unknown` in https://github.com/puppetlabs/facter/pull/2488/files#diff-04a930160736c529c24a312e18234bb563c12e7d946c9e084ae3cd499260d2a7R273

Reply:
Ah either the expected value for [`ens160`](https://github.com/puppetlabs/facter/pull/2488/files#diff-04a930160736c529c24a312e18234bb563c12e7d946c9e084ae3cd499260d2a7R286) needs to be `unknown` or the stub needs to [return `up`](https://github.com/puppetlabs/facter/pull/2488/files#diff-04a930160736c529c24a312e18234bb563c12e7d946c9e084ae3cd499260d2a7R301)

Reply:
Oops, I was looking at the wrong stanza :)

Should be fixed now.